Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/Drawables/InkDrawable.cs

```diff
@@ -27,6 +30,10 @@ internal class InkDrawable : IDrawable
 
         internal static readonly InkStrokeBuilder StrokeBuilder = new InkStrokeBuilder();
 
+        public InkDrawable()
+        {
+        }
```

Comment:
The IDrawableConverter's Read method has a hard reference to this method, so I thought a comment would be redundant. Yes, this is used by the deserialization of the json files, but thru the discriminator of the "$type", and I had to manually do this process (https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/043edd61d3d4e9eac6174b113580c5f6a31c6e56/Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/JsonConverters/IDrawableConverter.cs#L47-L54). Once System.Text.Json supports polymorphic deserialization (https://github.com/dotnet/runtime/issues/30083), then it would be thru reflection, and a comment would make sense. I can add it now, just in case.

Reply:
Oh I see, yeah sorry I missed that reference there, makes perfect sense! 👍